README.md

```diff
@@ -4,11 +4,50 @@ This is the userspace API for the robot, which is what students will interface w
 
 # How to use
 
-Currently the code is designed to be imported then ran, however we hope to move the robot initialisation into the __init__.py to not need this
+Currently the code is designed to be imported then ran, however we hope to move the robot initialisation into the `__init__.py` to not need this
+
+Markers can be seen by:
+
+``` python
+markers = r.cameras[0].see()
+# or
+markers = r.camera.see()
+```
+
+Servo positions can be set by:
 
 ``` python
-from robot import Robot
-robot = Robot() # TODO move this to the __init__.py so students don't need to do it
+r.servo_boards[0].ports[0].position = -1
+# or
+r.servo_board.ports[0].position = -1
+```
+
+Motors can be set with:
 
-robot.motor_boards[0].m0.voltage = 1
+``` python
+r.motor_boards[0].m0.voltage = -1
+# or
+r.motor_board.m0.voltage = -1
 ```
+
+The zone the robot is going to start in can be gotten with:
+
+``` python
+r.zone
+# or
+r.game.zone
+```
+
+# Testing
+
+the robot-api tests require `robotd` (available at [https://github.com/sourcebots/robotd] ) in the python path (ie in the same directory as the `tests` folder ) Personally I create a soft link to the robotd directory
+
+to run the tests, simply run `nosetests` (requires the `nose` python package) and it will run all tests
+
+
+# Zone ID script
+
+To run the zone script on USB insert, you must install the runusb script in [https://github.com/sourcebots/runusb]
+
+The zone ID script can be found at `robot/zone_script.py`, to use this for real robots you must put that script in a USB stick
+In the same directory as a file named `zone-<x>`, where `<x>` is the ID of the zone the robot should be with.
```

Comment:
Fixed this in 379e3d443ddc0419366b12834d1fd69bf4cfc0a2 then re-fixed it in db54a3be4575421a0fdae4affc013d62cf0d375e, and added full stops in 4ae00073df21a1270ba1bec19f360b07ffef7e0f